lib/workers/repository/process/lookup/generate.ts

```diff
@@ -18,19 +18,26 @@ export function generateUpdate(
   const update: LookupUpdate = {
     bucket,
     newVersion,
-    newValue: null,
+    newValue: null!,
```

Comment:
If this code would be easier to refactor, it already would be done I believe.

Since this is core functionality and we mostly don't want any changes to its behavior, I propose to manage our expectations for this strict-nullable *last mile*, so to speak. 

Let's just make code flaws explicit for now, but once we've enabled strict null checks project-wide, we'll have some good tooling at our disposal.